src/assets/custom/js/websiteHeader.js

```diff
@@ -1,19 +1,19 @@
 var websiteHeader = function () {
-  var HEADER_SELECTOR = ".website-header";
-  var REVEALED_CLASS = "website-header--revealed";
-  var HEADER_HAS_OPEN_SUBMENU_CLASS = "website-header--has-open-submenu";
-  var tolerance = 2;
+  window.__CODURANCE = window.__CODURANCE || {};
+  window.__CODURANCE.websiteNavigation = __CODURANCE.websiteNavigation || {};
 
-  var previousWindowPosition;
+  var closeSubMenu;
```

Comment:
Why is this hear? Why not move it closer to where it's used

Reply:
This var has been removed now since it's not needed. https://github.com/codurance/site/pull/1940/commits/bbea935a1c2a48d0f6f53ee09248e47647187d59